src/components/Search.js

```diff
@@ -1,22 +1,44 @@
-import React from 'react';
+import React from "react";
+import Button from "./Button.js";
 
-const Search = props => (
-  <div className="search">
-    <div className="page-header">
-      <h4 className="text-left">Search Bookings</h4>
-    </div>
-    <div className="row search-wrapper">
-      <div className="col">
-        <div className="form-group search-box">
-          <label htmlFor="customerId">Customer Id</label>
-          <div className="search-row">
-            <input type="text" id="customerId" className="form-control" placeholder="Customer Id" />
-            <button className="btn btn-primary fn-submit-name">Search IDs</button>
+export default class Search extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: ""
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
```

Comment:
Remember in class how we talked about using this syntax:

```js
handleChange = (event) => {
  // ...
}
```

If you use that, then you don't need to add the line in the constructor with `.bind(this)` in it.